docs/file_import_parameters.rst

```diff
@@ -0,0 +1,77 @@
+File import parameters
+======================
+
+``file-like`` parameters specify where to read the input data from. They can
+be:
+
+* a file name.
+* a file object, representing a file opened in binary mode.
+* an object that acts like an instance of `io.BufferedIOBase`_. Reading from it
+  returns bytes.
+
+``format`` is a string specifying an input format.
+The following input formats are supported:
+
+* "msgpack" - the data is MessagePack_ serialized
+* "json" - the data is JSON_ serialized.
+* "csv" - the data is CSV, and will be read using the `Python CSV module`_.
+* "tsv" - the data is CSV, with ``dialect=csv.excel`` explicitly set, and will
```

Comment:
I think "tsv" means "Tab Separated Values", is it correct to mention as "CSV"?

Reply:
Ah - thank you - I documented that wrong! The dialect chosen is csv.excel_tab. I shall rewrite the documentation to correct that and make it clear. Many thanks for spotting that, as it was very misleading.

Reply:
I've now fixed that.